vyper/ast/nodes.py

```diff
@@ -606,6 +606,7 @@ class arg(VyperNode):
 
 class Return(VyperNode):
     __slots__ = ('value', )
+    _is_terminus = True
```

Comment:
Is this explained anywhere like the others?

Reply:
No, good call.  Will document.